main.go

```diff
@@ -127,8 +128,10 @@ func (p DockerAuthZPlugin) evaluate(ctx context.Context, r authorization.Request
 		log.Printf("Returning OPA policy decision: %v (error: %v; input: %v)", allowed, err, i)
 	} else {
 		log.Printf("Returning OPA policy decision: %v", allowed)
-		dl, _ := json.Marshal(decision_log)
-		log.Println(string(dl))
+		if !p.quiet {
+			dl, _ := json.Marshal(decision_log)
+			log.Println(string(dl))
+		}
```

Comment:
I'd probably suppress both log messages if -quiet is given. The first log message is not going to be of much use without the other information (e.g,. users would just see `Returning OPA policy decision: false` which provides little context to help understand the decision.)

Reply:
OK. I understand your point. What about other logging messages available in the code?

For example:

- `log.Printf("OPA policy file %s does not exist, failing open and allowing request", p.policyFile)`
- `log.Printf("Returning OPA policy decision: %v (error: %v; input: %v)", allowed, err, i)`

Shall we supress all if `-quiet` is given?

Reply:
I'd just suppress the logs that happen on every check during normal operation. I'd leave the "file does not exist" log alone because that shouldn't happen and indicates an error.

Reply:
I've included the `log.Printf("Returning OPA policy decision: %v", allowed)` message as well. All other messages are still logged as they're erratic of some sort.